shake256-avx2/f1600x4.s

```diff
@@ -0,0 +1,901 @@
+# Generated by PeachPy 0.2.0 from test.py
+
+
+#ifdef __APPLE__
+.section __TEXT,__text,regular,pure_instructions
+.globl _f1600x4AVX2
+.p2align 4, 0x90
+_f1600x4AVX2:
+#else /* !__APPLE__ */
+.text
+.p2align 4,,15
+.globl f1600x4AVX2
+.type f1600x4AVX2, @function
+f1600x4AVX2:
+#endif /* !__APPLE */
+	movq $6, %rax
+17: # loop.begin:
+		vmovdqa 0(%rdi), %ymm8
```

Comment:
This doesn't compile with MSVC

Reply:
I don't have a windows machine available to me.  What is the error?

Reply:
MSVC has completely different syntax; there is not a chance to write assembly code that works for both Windows and gnu-style assemblers without going to a portable assembler like NASM.

Reply:
peachpy (which I used to generate the assembler, see `f1600x4.py`) does support nasm as output.

Reply:
Could you not generate microsoft assembly as `.asm` files? Then those could be used on Windows (with a `nmake` `Makefile` (see PQClean) or you could consider contributing an appropriate `CMakeList.txt`).

Reply:
Unfortunately peachpy doesn't support microsoft asm.  It can directly generate the windows object file.  It can also generate nasm.  What would we prefer?  @joostrijneveld